build_web_compilers/build.yaml

```diff
@@ -11,20 +11,31 @@ builders:
         - .ddc.js.map
     is_optional: True
     auto_apply: all_packages
-    required_inputs: [".dart"]
+    required_inputs:
+      - ".dart"
   entrypoint:
     target: "build_web_compilers"
     import: "package:build_web_compilers/builders.dart"
-    builder_factories: ["webEntrypointBuilder"]
+    builder_factories:
+      - "webEntrypointBuilder"
     build_extensions:
       .dart:
         - .dart.bootstrap.js
         - .dart.js
         - .dart.js.map
-    required_inputs:  [".dart", ".ddc.js", ".module"]
+    required_inputs:
+      - .dart
+      - .ddc.js
+      - .module
     build_to: cache
     auto_apply: root_package
     defaults:
       generate_for:
-        include: ["web/**", "test/**_test.dart"]
-        exclude: ["test/**.node_test.dart", "test/**.vm_test.dart"]
+        include:
+          - web/**
+          - test/**_test.dart
+          - example/**
```

Comment:
I'm a bit concerned about this change. It means we'll spam errors when there are non-web entrypoints in example/

Reply:
Only if there are non-web examples in example, and you have a dependency on build_web_compilers. The idea is just to try and do the right most of the time - I think its fairly safe to assume web entrypoints everywhere if somebody has this dependency.